src/ucp/core/ucp_worker.c

```diff
@@ -609,6 +609,113 @@ ucs_status_t ucp_worker_set_ep_failed(ucp_worker_h worker, ucp_ep_h ucp_ep,
     return ret_status;
 }
 
+const void *ucp_worker_ep_match_get_address(const ucs_conn_match_elem_t *conn_match)
```

Comment:
maybe keep the ep_match.c/h files, and put the wrapper functions there?

Reply:
> maybe keep the ep_match.c/h files, and put the wrapper functions there?

@yosefe yes, it's better

Reply:
@yosefe fixed